### ISSUE TITLE
mSubCube -p not recognized, and then doesn't allow a 1x1 region

### ISSUE THREAD

**Issue by teuben** [Caltech-IPAC/Montage#26]
Two very strange issues:
1) mSubCube -p ngc6503.fits junk1.fits 197 98 1 1

        [struct stat="ERROR", msg="Invalid flag "-p"."]

2) /astromake/opt/montage/git/Montage/mSubCube -p -d ngc6503.fits junk1.fits 197 98 2 2

this works, but creates a 3x3 region.  I would like to argue it's perfectly legal to get a 1x1 region, as the pixel size gives the size.

The other question is :  why does the executable in the bin not work,and why do I find two sources mSubCube.c  and two binaries????

**Comment by teuben** [comment]
I solved the 1x1 construction, and will put that in a pull request after some more consultation

**Comment by JohnGood** [comment]
While we have maintained a presence for Montage on GitHub for a several years now,
until recently the actual source code control and issue tracking were done through
other, institution-specific, tools.  So while we addressed all issues in a timely
manner, we were regrettably lax about cleaning up these issues in GitHub.

There was a pull request that went along with this issue and that was processed
at the time.
